release/charts/charts.go

```diff
@@ -0,0 +1,57 @@
+package charts
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/rancher/ecm-distro-tools/cmd/release/config"
+)
+
+// List prints the lifecycle status of the charts
+func List(ctx context.Context, c *config.ChartsRelease, branch, chart string) (string, error) {
+	var branchArg, chartArg string
+
+	branchArg = "--branch-version=" + branch
+	if chart != "" {
+		chartArg = "--chart=" + chart
+	}
+
+	output, err := runChartsBuildScripts(c.Workspace, "lifecycle-status", branchArg, chartArg)
+	if err != nil {
+		return "", err
+	}
+
+	response := string(output) + fmt.Sprintf("\ngenerated log files for inspection at: \n%s\n", c.Workspace+"/logs/")
+	return response, nil
+}
+
+func runChartsBuildScripts(chartsRepoPath string, args ...string) ([]byte, error) {
```

Comment:
I thought I had left a comment about changing the name but I guess not. Please change this to be runChartsBuild